curriculum/challenges/english/25-front-end-development/lab-recipe-page/668f08ea07b99b1f4a91acab.md

```diff
@@ -89,19 +89,19 @@ assert.lengthOf(document.querySelectorAll('h1'), 1)
 You should have a `p` element below your `h1` element with text describing your recipe.
 
 ```js
-assert($('p')[0].previousElementSibling.tagName === 'H1');
+assert.strictEqual(document.querySelector('p')?.previousElementSibling.tagName === 'H1');
```

Comment:
```suggestion
assert.strictEqual(document.querySelector('p')?.previousElementSibling.tagName, 'H1');
```
You're close, but the `strictEqual` method takes two arguments to compare, not a conditional statement.